src/commands/trace/trace.ts

```diff
@@ -101,6 +104,7 @@ export class TraceCommand extends Command {
           error_message: stderr,
           exit_code: exitCode,
           is_error: exitCode !== 0,
+          measures,
```

Comment:
are we OK with this sending null measures? If the user sets a string measure `parseFloat` would return `NaN` which is serialized as `null` by `JSON.stringify`: 
```
> JSON.stringify({ a: parseFloat('a')})
'{"a":null}'
```

Reply:
I tried that and it ends up as a 0 after all the processing. I'd say it's an ok-ish behavior, but maybe it's better to fail the command? What do you think?

Reply:
I don't think it's worth failing the command, but a `0` doesn't seem correct either. Maybe we can do: 

```javascript
Object.fromEntries(
    Object.entries(cliMeasures)
        .map(([key, value]) => [key, parseFloat(value)])
        .filter(([key, value]) => {
            if (isNaN(value)) {
                logger.debug(`Value for ${key} is not a number`);
                return false;
            }
            return true;
        }),
);
```
WDYT? Maybe even without the logger.